lib/request-preview-helpers.rb

```
@@ -0,0 +1,235 @@
+            comma = index < params.size - 1 ? "," : ""
+            text << "        \"#{key}\": #{value.inspect}#{comma}"
+        end
+
+        text << "    }' \\"
+    end
+
+    # Remove the trailing backslash from the last line
+    text[-1] = text[-1].chomp(" \\")
+
+    "```shell\n#{text.join("\n")}\n```"
+end
```

RuboCop (1.81.7)

[convention] Assignment Branch Condition size for `shell_preview` is too high. [<10, 19, 20> 29.34/23]

(Metrics/AbcSize)